site/content/docs/developing/domain.en.md

```diff
@@ -30,4 +22,14 @@ http:
   alias: example.aws
 ```
 
-Note that for now you can only use aliases under the domain you specified when creating the application. We'll make this feature more powerful by allowing users to import certificates so as to use any alias in the future!
+!!!info
+    Note that for now you can only use aliases under the domain you specified when creating the application. We'll make this feature more powerful by allowing users to import certificates so as to use any alias in the future!
+
+## What is Copilot doing when using domain?
```

Comment:
Sorry I missed this in last round! Maybe something like "What happens under the hood?" like in https://aws.github.io/copilot-cli/docs/commands/storage-init/#what-happens-under-the-hood.